spec/spec_helper.rb

```diff
@@ -1,7 +1,11 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
+require 'combustion'
 require 'stitch_fix/log_weasel'
+
+Combustion.initialize! :active_support
```

Comment:
I read the README it doesn't explain exactly what it does.

So…reading the source, I can answer my own question, which is that this gem causes the initializer to execute.  Before, it wasn't, so we didn't know that the initializer would blow up.

Now, we are executing it, but only testing that the code within doesn't blow up (you can hard-code any value in there and the tests still pass).  Likely we still need some sort of test coverage for the `||` logic inside there, but this gem seems useful insomuch as it executes the code in a "real-life" sorta way. (note that we don't need to pass any arguments to `initialize!` - this issue doesn't have anything to do with active support.  Omitting the arg still executes the initializer)